configure.ac

```diff
@@ -827,6 +844,7 @@ else
     EXTRA_CFLAGS="-O2 -g -Wall $strict_compile"
   fi
 fi
+EXTRA_CFLAGS="$EXTRA_CFLAGS $assertions"
```

Comment:
I'm afraid this has no effect. I mean I run `./configure ... --enable-assertions`, but I don't see the `DEBUG` flag at all in generated files (`Makefile`):
```
$ grep -rI MODSEC_EXTRA_CFLAGS * | grep DEBUG
[EMPTY]
```

Reply:
--disable-assertions => NDEBUG
--enable-assertions => nothing special